bazel/utils/container/container.bzl

```diff
@@ -120,6 +120,10 @@ def container_image(*args, **kwargs):
     output = "{}_labels.txt".format(name)
     tags = kwargs.get("tags", [])
 
+    # Remove once oci_pull doesn't have auth errors anymore
+    if kwargs.get("base", "").startswith("@"):
```

Comment:
I don't understand this bit.

Reply:
Ok, updated the comment with a better explanation.